src/parser.c

```diff
@@ -3160,42 +3160,43 @@ static rbs_comment_t *comment_get_comment(rbs_comment_t *com, int line) {
 }
 
 static void comment_insert_new_line(rbs_allocator_t *allocator, rbs_comment_t *com, rbs_token_t comment_token) {
-    if (com->line_count == 0) {
```

Comment:
By changing `alloc_comment()` to not call `comment_insert_new_line()`, we never have to deal with the 0 case here 🥳